nexus-repository-apk-it/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>apk-parent</artifactId>
-    <version>0.0.18-SNAPSHOT</version>
+    <version>0.0.19-SNAPSHOT</version>
```

Comment:
You should be able to revert this change, and the "maven release" process takes care of releasing version `0.0.18` and incrementing the development version to the next revision (`0.0.19-SNAPSHOT`).